src/core/crypto.js

```diff
@@ -1,11 +1,15 @@
-
 import crypto from 'crypto';
 
 const algorithm = 'aes-256-cbc';
 
-export const DEFAULT_KEY = process.env.ENCRYPTION_KEY || process.env.REACT_APP_ENCRYPTION_KEY;
+export const DEFAULT_KEY =
+  process.env.ENCRYPTION_KEY || process.env.REACT_APP_ENCRYPTION_KEY || '776fe87eec8c9ba8417beda00b23cf22f5e134d9644d0a195cd9e0b7373760c1';
```

Comment:
the linter on my machine wanted this line split up, but maybe I don't have the right settings yet

Reply:
seems fixed